src/pages/projectView/projectMainBody/components/ProjectRewardPanel.tsx

```diff
@@ -30,11 +30,17 @@ export const ProjectRewardPanel = ({ reward }: Props) => {
   }
 
   const renderRewardAvailability = () => {
-    if(rewardStockRemaining > 0) {
-      return <><Box as={'span'} color={'secondary.red'}>{rewardStockRemaining + ` ${t('remaining')}`}</Box> <Box as={'span'} style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</Box> </>;
-    } else if (rewardStockRemaining === 0) {
+
+    if(rewardStockRemaining === 0) {
       return <><Box as={'span'} color={'neutral.600'} fontWeight={700}>{t('Sold Out')}</Box> <Box as={'span'} style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</Box> </>;
-    } else {
+    } 
+    else if ( rewardStockRemaining > 3 ) {
+      return <><Box as={'span'}>{rewardStockRemaining + ` ${t('remaining')}`}</Box> <Box as={'span'} style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</Box> </>;
+    } 
+    else if ( rewardStockRemaining > 0 && rewardStockRemaining <= 3 ) {
+      return <><Box as={'span'} color={'secondary.red'}>{rewardStockRemaining + ` ${t('remaining')}`}</Box> <Box as={'span'} style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</Box> </>;
+    }
+    else {
```

Comment:
I mentioned this last time too, this UI and the UI in src/components/molecules/projectDisplay/RewardCard.tsx

are literally the same, it could be a separate component, that is only dependent on the `rewardStockRemaining` value.

That way, when there are future changes, it'll be only in one place, like right now.